templates/telemetry/heartbeat/heartbeat.4.schema.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "heartbeat",
+  "type": "object",
+  "properties": {
+    @TELEMETRY_APPLICATION_1_JSON@,
+    @TELEMETRY_CLIENTID_1_JSON@,
+    @TELEMETRY_CREATIONDATE_1_JSON@,
+    @TELEMETRY_ID_1_JSON@,
+    "payload": {
+      "additionalProperties": false,
+      "properties": {
+        "version": {
+          "maximum": 1,
+          "minimum": 1,
+          "type": "number"
+        },
+        "flowId": {
+          @COMMON_PATTERN_UUID_1_JSON@,
+          "type": "string"
+        },
+        "offeredTS": {
+          "minimum": 0,
+          "type": "integer"
+        },
+        "learnMoreTS": {
+          "minimum": 0,
+          "type": "integer"
+        },
+        "votedTS": {
+          "minimum": 0,
+          "type": "integer"
+        },
+        "engagedTS": {
+          "minimum": 0,
+          "type": "integer"
+        },
+        "closedTS": {
+          "minimum": 0,
+          "type": "integer"
+        },
+        "expiredTS": {
+          "minimum": 0,
+          "type": "integer"
+        },
+        "windowClosedTS": {
+          "minimum": 0,
+          "type": "integer"
+        },
+        "score": {
+          "minimum": 1,
+          "type": "integer"
+        },
+        "surveyId": {
+          "type": "string"
+        },
+        "surveyVersion": {
+          "pattern": "^[a-fA-F0-9]{64}$",
+          "type": "string"
+        },
+        "testing": {
+          "maximum": 1,
+          "minimum": 1,
+          "type": "integer"
```

Comment:
`testing` is apparently 1 or null according to https://github.com/mozilla/normandy/blob/master/recipe-server/client/actions/show-heartbeat/index.js#L272